src/citation-transformer.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * © 2025 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// import { Citation } from "@citation-js/core"
+import * as Citation from '@citation-js/core'
+import { BibliographyItem } from '@manuscripts/json-schema'
+
+const loadCitationPlugins = async () => {
+  await import('@citation-js/plugin-bibtex')
```

Comment:
Please user Promise.all, using await after await will create a cascade of requests where no other requests will be started until previous is completed.